flow-defs/checkbox.js.flow

```diff
@@ -8,10 +8,12 @@ declare type RenderProps = {
   checked?: boolean,
   onChange?: (value: boolean) => void,
   id?: string,
-  render?: (
-    checkboxElement: React.Element<any>,
-    labelId: string
-  ) => React.Element<any>,
+  render?: (renderProps: {
+    iconElement: React.Element<any>,
```

Comment:
checkboxElement? to me the icon name is misleading

Reply:
I don't like `checkboxElement`, because it's not the checkbox. For me, the checkbox element is the element with the `role="checkbox"`, and this is just the checkbox icon.
About `actionElement`, hmm... not sure if it's better or worse

Reply:
controlElement ?